[CICD] app이 redis에 접근 못함

## 개요
- redis는 실행 중인데, app이 redis에 접근을 못해서 죽어버림

## 상세 설명
- 에러 로그
```
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'rateLimitService' defined in URL [jar:nested:/app/app.jar/!BOOT-INF/classes/!/com/codeit/playlist/domain/auth/passwordratelimit/RateLimitService.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'redissonClient' defined in class path resource [com/codeit/playlist/global/config/RateLimitConfig.class]: Failed to instantiate [org.redisson.api.RedissonClient]: Factory method 'redissonClient' threw exception with message: java.util.concurrent.ExecutionException: org.redisson.client.RedisConnectionException: java.net.UnknownHostException: Failed to resolve 'playlist-redis' [A(1)]
```

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -83,7 +83,7 @@ jobs:
                 {"name": "LOGS_BUCKET", "value": $LOGS_BUCKET},
                 {"name": "PROFILE_BUCKET", "value": $PROFILE_BUCKET},
                 {"name": "CONTENT_BUCKET", "value": $CONTENT_BUCKET},
-                {"name": "REDIS_HOST", "value": "playlist-redis"},
+                {"name": "REDIS_HOST", "value": "localhost"},
                 {"name": "REDIS_PASSWORD", "value": $REDIS_PASSWORD},
                 {"name": "REDIS_PORT", "value": "6379"},
                 {"name": "SPRING_MAIL_HOST", "value": $SPRING_MAIL_HOST},

